src/tests/ftest/util/soak_utils.py

```diff
@@ -475,14 +589,13 @@ def launch_extend(self, pool, name, results, args):
 
     if self.selected_host:
         ranklist = self.server_managers[0].get_host_ranks(self.selected_host)
-
+        ranks = ",".join(str(rank) for rank in ranklist)
```

Comment:
Each rank will already be a `str` because it's a `NodeSet`, so FYI
```suggestion
        ranks = ",".join(ranklist)
```